test/fika/parser_test.exs

```diff
@@ -27,6 +27,32 @@ defmodule Fika.ParserTest do
     end
   end
 
+  describe "atom" do
+    test "parses single-char atom" do
+      # assert that for each letter in the alphabet,
+      # we generate a valid atom
+      0..25
+      |> Enum.each(fn offset ->
+        # Trick to offset from a to z
+        c = [hd('a') + offset]
+        # charlists are interpolated as strings, to offset = 1 yields 'b'
+        # which in turn is interpolated to yield str = ":b"
+        atom = :"#{c}"
+        str = ":#{atom}"
+
+        assert Parser.expression!(str) == {:atom, {1, 0, 2}, atom}
+      end)
+    end
```

Comment:
Why do we test separately for single character atoms? If multi-char atoms work, single-char atoms should work too, I suppose?

Reply:
I took this route because this depends on the way the code treats it. So if there were any off-by-on errors, this would catch them. However, if you see fit, I could just remove either one of the tests.

Reply:
Got it. Let's keep the multi-char test and remove the single-char one in that case.